war_and_peace_runner.rb

```diff
@@ -0,0 +1,38 @@
+require 'pry'
+require './lib/card'
```

Comment:
Make sure to get rid of any unnecessary `require`'s when turning in your submission. And keep in mind, since we are not "new-ing up" any `Card` objects inside this runner file, the `require './lib/card'` line can also be removed.